packages/fonts/src/font-data.ts

```diff
@@ -41,32 +22,47 @@ export const parseSubfamily = (subfamily: string) => {
       break;
     }
   }
-  let weight: WeightValue = weights.normal;
-  let possibleWeight: WeightKey;
-  for (possibleWeight in weights) {
-    if (subfamilyLow.includes(possibleWeight)) {
-      weight = weights[possibleWeight];
+
+  let weight: FontWeight = "400";
+  for (weight in fontWeights) {
+    const { name } = fontWeights[weight];
+    const { alt } = fontWeights[weight];
+    if (subfamilyLow.includes(name) || subfamilyLow.includes(alt)) {
       break;
     }
   }
-  return { style, weight };
+  return { style, weight: Number(weight) };
+};
+
+export const normalizeFamily = (family: string, subfamily: string) => {
```

Comment:
I couldn't understand what this function does 😅

Reply:
explained, helped?

Reply:
Yes, but the logic inside the function seem very unpredictable. For example: `normalizeFamily("Robolder Bold", "Bold")` will produce `"Roer Bold"`

Reply:
you are right, I was operating on a subset of values

Reply:
Still feels kinda unpredictable. Is it required that `normalizeFamily("Roboto Bold", "Bold") === normalizeFamily("Roboto Black", "Black")`? If yes, how do we know user won't upload some weird font for which this will not be the case?

Reply:
Also, why do we assume they will be split by " "? What if it `normalizeFamily("RobotoBold", "Bold")`.

Maybe I'm overthinking it, and the output of this function is not important. I don't understand well where the result passed down the line and what the requirements for it.

Reply:
> Also, why do we assume they will be split by " "? What if it normalizeFamily("RobotoBold", "Bold").

In that case we don't care, the font name is RobotoBold then, its weird but its the name. What we are trying to separate is the weight keyword from the font name, which happens to be the case most of the time in font naming.

Reply:
> Is it required that normalizeFamily("Roboto Bold", "Bold") === normalizeFamily("Roboto Black", "Black")
Yes we kind need to let user use "Roboto" and bold is the weight we allow to select separately. Same font name, different weights

Reply:
> If yes, how do we know user won't upload some weird font for which this will not be the case?

We normalize what we can normalize, if we can't - it will be displayed as a standalone separate font, user can still use it, but its a separate font name.

Reply:
Ok, got it 👍